integration_tests/e2e/deactivate/permanent.cy.ts

```diff
@@ -372,14 +375,59 @@ context('Deactivate permanent', () => {
         cy.get('.change-summary p').contains(/The establishment’s maximum capacity will reduce from 10 to 8.$/)
       })
 
-      it('shows the success banner on completion', () => {
-        const confirmationPage = Page.verifyOnPage(DeactivatePermanentConfirmPage)
-        confirmationPage.confirmButton().click()
+      context('With the VIEW_INTERNAL_LOCATION role', () => {
+        let locations: Location[]
```

Comment:
My IDEA complains about this because it doesn't recognise that `LocationFactory.build` returns a `Location`, can we do this instead please?

```suggestion
        let locations: ReturnType<typeof LocationFactory.build>[]
```